api/tests/python/models/test_request.py

```diff
@@ -1,5 +1,8 @@
 from flask import jsonify
 from unittest import mock
+from unittest.mock import patch
+from namex.utils import queue_util
+from flask import current_app
 import pytest
```

Comment:
this will fail linting as it doesn't support PEP styling (base, then 3rd party, then local) imports
```suggestion
from unittest import mock
rom unittest.mock import patch

import pytest
from flask import current_app
from flask import jsonify

from namex.utils import queue_util
```
You can also put multiple imports on the same line if you want.

Reply:
dont fully understand this, but i implemented your suggestions, and repushed

Reply:
In our _lint_ setup, we use a package called _isort_
it has rules about how imports are laid out, sort order, etc. The pattern is sorted Top->Bottome, Left-Right alphabetically descending, and then grouped together:
import core packages from python
space
import 3rd party packages
space
import from this package